### ISSUE TITLE
Add support for page-foreground-image

### ISSUE THREAD

**Issue by benlandg** [Mogztter/asciidoctor-web-pdf#701]
Currently Asciidoctor-web does not seem to support :page-foreground-image: -attribute. 
Would be great to include it.

**Comment by ggrossetie** [comment]
Indeed, you can configure a foreground image using CSS but we do not support the attribute. Feel free to open a pull request to implement it.
